server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts

```diff
@@ -58,7 +58,9 @@ export class RiddlerResourcesFactory implements IResourcesFactory<RiddlerResourc
         let globalDbMongoManager;
         const globalDbEnabled = config.get("mongo:globalDbEnabled") as boolean;
         if (globalDbEnabled) {
-            globalDbMongoManager = new MongoManager(factory, false, null, true);
+            // TODO, this is an experimental flag that try to see if we could have mongo connection error fixed.
```

Comment:
I think we can probably remove these comments (here and in the other files). Having `globalDbReconnect` is probably a good thing to have, so even if it does not fix the Mongo Connection Error, it might still be useful later.

Reply:
I think based on previous db effort, mongo team suggest to not do pool level reconnect but relays on connection reconnection inside the pool.
This reconnect is more like experiment bandit. We should remove this once we figure out what is going with connection retry.

Reply:
I think the TODO related to whether we want it to be true or false applies to FRS specifically, but not OSS. This is just a config and does not change the behavior in this OSS repo - it will still be false because you did not override the values in config.json or fluid-configmap.json.

However, if we still want to keep this TODO, then we would probably want to create a GitHub issue instead. That has been the guidance from the FF team. But then again, the Issue would more appropriate for FRS and not OSS, so it might not apply here.

Reply:
I see what you mean, removed the in 94219689fa8515d9ab38aa7319683b29a91e208c